ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/common/utils/LocaleUtilsTest.java

```diff
@@ -0,0 +1,44 @@
+package com.kayako.sdk.android.k5.common.utils;
+
+import com.kayako.sdk.helpcenter.locale.Locale;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class LocaleUtilsTest {
+    private static final String LOCALE_SPAIN = "es-ES";
```

Comment:
✅ Good to see constant used instead of hard-coded values.

This approach increases maintainability (single point of modification) and debugging, avoid duplication and decreases verbosity.

**No change suggested, looks good!**	  

<a id="da48d530-48ef-11e8-90f2-2f455003fed5" href="none" codereview-uuid-reserved-tag />